### PR TITLE
Extend webpack conf

### PR DESCRIPTION
**Pull request by titkov49**
This is needed in order to get rid of webpack configuration for monaco editor in products where it is not used, but keep it in other repos where it's required.
Also `LibraryModule` has been removed.

### CODE REVIEW

**Comment by titkov49** [comment]
Documentation is in progress

File: webpack/webpack.build.js [hunk 1]
```diff
@@ -7,9 +7,26 @@ const InterpolateHtmlPlugin = require('interpolate-html-plugin');
...
+	extraPlugins: [],
+	optimization: {},
+	module: {
+		extraRules: []
+	}
+};
+
+// load extended configuration
+try {
+	extendedConfig = { ...extendedConfig, ...require("../../asab-webui.config") } ;
```
Comment (Pe5h4) [inline webpack/webpack.build.js:25]: Can you please add comment why you have a relative path here?

File: webpack/webpack.dev.js [hunk 2]
```diff
@@ -4,10 +4,27 @@ const path = require('path');
...
+	extraPlugins: [],
+	optimization: {},
+	module: {
+		extraRules: []
+	}
+};
+
+// load extended configuration
+try {
+	extendedConfig = { ...extendedConfig, ...require("../../asab-webui.config") } ;
```
Comment (Pe5h4) [inline webpack/webpack.dev.js:22]: Can you please add comment why you have a relative path here?

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Seems fine to me, I miss some comment about why the config is taken as relative path

File: webpack/webpack.dev.js (hunk 2 above)
Comment (titkov49) [reply webpack/webpack.dev.js:24]: @Pe5h4 Hi Franta,
I actually do not understand what do you mean by why do I have it there

**Review by ateska — APPROVED** [review]

**Review by Pe5h4 — APPROVED** [review]

**Pull request merged**